src/xcode/ENA/ENA/Source/Services/Exposure Transaction/ExposureDetection.swift

```diff
@@ -47,70 +48,22 @@ final class ExposureDetection {
 		progress?.cancel()
 	}
 
-	private func getSupportedCountries(completion: @escaping ([Country]) -> Void) {
-		delegate?.exposureDetection(supportedCountries: { [weak self] result in
-			guard let self = self else { return }
-
+	private func downloadKeyPackages(completion: @escaping () -> Void) {
+		self.countryKeypackageDownloader.downloadKeypackages(for: country) { result in
```

Comment:
I like `self` (not 'myself'), but is this needed here?

```suggestion
		countryKeypackageDownloader.downloadKeypackages(for: country) { result in
```

Reply:
No... good eyes! It's a leftover. The code was in a closure before. I'll clean this up.